app/lib/common/pages/medications/cubit.dart

```diff
@@ -13,18 +13,43 @@ class MedicationsCubit extends Cubit<MedicationsState> {
   final int _id;
 
   Future<void> loadMedications() async {
-    final requestUri = annotationServerUrl.replace(
-      path: 'api/v1/medications/$_id',
-    );
     emit(MedicationsState.loading());
-    final response = await http.get(requestUri);
-    if (response.statusCode != 200) {
+    final response = await sendRequest();
+    if (response == null){
```

Comment:
yeah I have a code formatter, but I have automated file saving activated, thus in my mind it runs sort of randomly. for example right now the changes you proposed already got applied automatically, but I dont really understand why they were not applied before commiting. I'll look into VSCode's settings there to avoid these unnecessary mistakes

Reply:
I think its fine with the return statement :)